src/main/java/com/zendesk/maxwell/bootstrap/SynchronousBootstrapper.java

```diff
@@ -150,7 +152,8 @@ public void resume(AbstractProducer producer, Replicator replicator) throws Exce
 			// This update resets all rows of incomplete bootstraps to their original state.
 			// These updates are treated as fresh bootstrap requests and trigger a restart
 			// of the bootstrap process from the beginning.
-			String sql = "update `bootstrap` set started_at = NULL where is_complete = 0 and started_at is not NULL";
+			String clientid = ((BinlogConnectorReplicator)replicator).getClientID();
+			String sql = "update `bootstrap` set started_at = NULL where is_complete = 0 and started_at is not NULL and client_id = '" + clientid + "'";
```

Comment:
this needs to use regular sql quoting via `client_id = ?` and `PreparedStatement#setString()`

Reply:
I fix the code according to review comments. But not sure why the build fails. Can You help?